### ISSUE TITLE
Remove audio stream by language transcodes audio stream to vorbis

### ISSUE THREAD

**Issue by tigerkzr** [Unmanic/unmanic#418]
Have you read the [Code of Conduct](CODE_OF_CONDUCT.md)? By filing an Issue, you are expected
to comply with it, including treating everyone with respect:
yes

Remember that these are some guidelines, use your best judgement.

### Description
I was using the "remove audio stream by language" plugin set to remove Russian and Ukrainian (ru,uk).  Removal worked, but it transcoded the remaining English stream from DTS-HS MA 2.0 to Vorbis 2.0.

### Steps to Reproduce

1. Load file with audio streams in language(s) to be removed
2. Use plugin "remove audio streams by language" with setting for languages to remove
3. Run workers with file and plugins

**Expected behavior:** [What you expect to happen]
Designated language streams are removed and remaining language streams are untouched

**Actual behaviour:** [What actually happens]
Remains audio stream was transcoded from DTS-DH MA 2.0 to Vorbis 2.0

**Reproduces how often:** [What percentage of the time does it reproduce?]
at least a dozen files with Russian and English streams were processed and all files removed Russian stream but transcoded English stream.

### Versions
Version: [0.2.4~7339f8b] in docker container on Ubuntu 23.04 server

The version can be found at the footer of the WebUI. Also, please include the OS details and what version you are running
(e.g. `cat /etc/os-release`).

### Installation Method Used
Docker

### Additional Information
Am going to try the remove audio/subtitle by language as soon as I can restore the affected files.  Will report back.

EDIT: The "Remove audio/subtitle by language" appears to function properly as long as I use three letter language codes.  It does nothing if I use the two letter codes.

Any additional information, configuration or data that might be necessary to
reproduce the issue.

**Comment by regularguy01** [comment]
yes its a search string you might need several.   https://www.loc.gov/standards/iso639-2/php/code_list.php
